Source/Core/Common/MemArenaAndroid.cpp

```diff
@@ -141,24 +100,10 @@ u8* MemArena::FindMemoryBase()
   const size_t memory_size = 0x400000000;
 #endif
 
-#ifdef _WIN32
-  u8* base = static_cast<u8*>(VirtualAlloc(nullptr, memory_size, MEM_RESERVE, PAGE_READWRITE));
-  if (!base)
-  {
-    PanicAlertFmt("Failed to map enough memory space: {}", GetLastErrorString());
-    return nullptr;
-  }
-  VirtualFree(base, 0, MEM_RELEASE);
-  return base;
-#else
-#ifdef ANDROID
   // Android 4.3 changed how mmap works.
   // if we map it private and then munmap it, we can't use the base returned.
   // This may be due to changes in them support a full SELinux implementation.
```

Comment:
This isn't your comment, but I believe this is should be "due to changes in them <ins>to</ins> support a full SELinux implementation".